App-Service-Environment-AzFirewall/azuredeploy.parameters.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
remove all parameters that have defaultValue (will be most of them after the template updates)